test/files/scalacheck/MutablePriorityQueue.scala

```diff
@@ -0,0 +1,102 @@
+import scala.collection.mutable.PriorityQueue
+import org.scalacheck._
+import Prop._
+import Arbitrary._
+
+object PQTest extends Properties("PriorityQueue") {
```

Comment:
I think this has to be `object Test` to get run properly.  I'm not very familiar with ScalaCheck, but the other tests invariably use just `object Test` as the name.